app/src/main/res/values/strings.xml

```diff
@@ -533,36 +533,31 @@
+        <item>How can I share the PDF files that are edited/created using the app?#####You can access and share the PDF files which you have edited/created directly from the app. To share the PDF files, \n1) Click the View Files option on the home page to view all the PDF files you have created. \n2) Select the file(s) you want to share using the checkbox. \n3) Next, Click the More option (three vertical dots) at the top-right and click Share. \n4) Select the desired method to share the files.</item>
+        <item>Can I recover the forgotten password of a PDF file using the app?#####No, a lost password cannot be recovered.</item>
+        <item>Can I change the password of a PDF file?#####Yes, but only if you remove the existing password. Once the password is removed, use the Remove Passowrd option in the home page to change the password.</item>
+        <item>Do you collect any personal data when using the app?#####No, we do not collect any personal data.</item>
+        <item>Can I compress the images before creating a PDF?#####Yes, you can. To compress the images, \n1) Click the Images to PDF option on the homepage. \n2) Select the images using the SELECT IMAGES button. \n3) Next, select the Image compression option to compress the images.</item>
+        <item>What is the format of the images extracted from a PDF?#####The images extracted from a PDF will be in .png format.</item>
+        <item>Can I add files from cloud/online storage?#####No, you cannot as of now.</item>
+        <item>How to check history of files created/modified?#####To see all the files you have created/modified in the app. \nTo do this, click the History option on the home page.</item>
+        <item>How to merge multiple PDF files?#####To merge multiple PDFs, \n1) Click the Merge PDF option on the homepage. \n2) Select the PDF files using the SELECT FILES button. \n3) Once selected, click MERGE FILES button.</item>
+        <item>Can I edit images while creating a PDF?#####Yes, you can. To do so, \n1) CLick the Images to PDF option on the homepage. \n2) Select the images using the SELECT IMAGES. \n3) Once selected, click the Edit Images or Filter Images option based on your requirement. \n4) Finish editing or applying the filter. \n5) Now, select the edited images using the SELECT IMAGES button and click CREATE PDF button to create the PDF.</item>
+        <item>How to add watermark to the PDF?#####To add watermark scroll down to Enhance Created PDFs on the home page or in the options menu by swiping left to right and click on Add Watermark.</item>
```

Comment:
"Enhance Created PDFs"

Reply:
"Add Watermark"

Reply:
We don't have to use double quotation every time we mention a button name or element in the UI.

I suggest using bold instead of double quotation.

Reply:
I am happy with bold too.

Reply:
Just to confirm, I've used <strong>...</strong> tags for bold.